docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,182 @@
+
+Yo will be able to modify all of the field previously during the creation stage.
+
+To confirm the changes, click **'Save and update scenario'**. To cancel, click the **'Cancel'** button, both at the bottom of the modal.
+
+### Triggers
+
+Triggers describes situations in which tracking will be performed, such as button clicks, page views, app launches, in-app purchases, user interactions, login attemps, system events and more. These triggers are essential for developers when implementing tracking.  
+
+Add multiple triggers by clicking the **'+ Add trigger'** button on the **Scenario Triggers** section. A new modal view will appear with a text field where you can specificy information about when, where and how the scenario will be triggered.
+
+![](images/ts-6.png)
+
+Once completed, you can confirm the changes by clicking the **'Save and update scenario'** button or cancel the changes by clicking the **'Cancel'** button.
+
+You will be able to edit it after adding a trigger.
+
+### Implementation rules
+
+Implementation rules allow defining constraints and conditions on basic fields, based on the underlying schema properties of a particular tracking event. These rules ensure that the event data complies with the data structure's underlying schema, so when the scenario is triggered and flows downstream through the pipeline, it is validated against the schema.
+
+Add multiple implementation rules by clicking the **'+ Add implementation rules'** button in the **Implmentation rules** section. A new modal view will appear.
+
+This modal displays a list of rules for specific properties that can be set. This list of posible rules is populated based on the Data Structure properties from the specific version that the Tracking Scenario was created from. A search bar will also be available to help search and filtering by rule name.
+
+![](images/ts-7.png)
+
+You can click on any of the properties that are shown in the list. A new view inside the modal will show where you'll be able to configure the rule's **Type of condition**, **Value** and **Comments** properties. 
+
+![](images/ts-8.png)
+
+Click **'Add rule'** to add rules or **'Cancel'** button to discard changes.
+
+Once you've configured the rules, click the **'Save and update event'** to confirm the changes or **'Close'** to discard them.
+
+You will see a list of rules in the tracking detail view where you can edit or remove them.
+
+![](images/ts-9.png)
+
+### Entities
+
+Entities can be defined as part of the context for a tracking scenario, providing additional information about specific events such as clicks, conversions, impressions, payment info, and more. By including these entities, you can enrich the tracked data and gain deeper insights into user behavior.
+
+Add multiple entites by clicking the **'Add entities'** button. A new modal view for configuring entites will appear.
+
+![](images/ts-10.png)
```

Comment:
Should we mention something about the difference between source "Iglu central" and "User defined"? Currently it only mentions that they are sourced from customers Iglu